spec/features/user_invites_friend_spec.rb

```diff
@@ -1,11 +1,12 @@
 require 'spec_helper'
 
 feature 'user invites friend' do
-  scenario 'user succesfully invites friend and invitation is accepted' do
+  scenario 'user succesfully invites friend and invitation is accepted', { js: true, vcr: true } do
```

Comment:
Instead of hash like tags `{ js: true, vcr: true }` you can use array like symbols as tags `:js, :vcr` - its shorter :)